net-p2p/resilio-sync-bin/files/resilio-sync.initd

```diff
@@ -0,0 +1,41 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+extra_commands="reload"
+
+NAME=${SVCNAME##*.}
+if [ -n "${NAME}" -a "${SVCNAME}" != "resilio-sync" ]; then
+    PID="/run/resilio-sync/resilio-sync.${NAME}.pid"
+    PNAME=$(echo ${RC_SVCNAME} | sed 's/\..*//g')
+    CONF_DEFAULT="/etc/conf.d/resilio-sync.${NAME}"
+else
+    PID="/run/resilio-sync/resilio-sync.pid"
+    PNAME=${RC_SVCNAME}
+    CONF_DEFAULT="/etc/conf.d/resilio-sync"
+fi
+CONF=${CONF:-${CONF_DEFAULT}}
```

Comment:
Did you mean `CONFIG`? That's what conf.d has. But it has `--config` explicitly specified, so I'm confused.

Also, I think it's marginally better to do:

    : "${CONF:=${CONF_DEFAULT}}"